### PR TITLE
[MOSIP-31977] merge code from release-1.2.0.1 to develop

### PR DESCRIPTION
**Pull request by mahammedtaheer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[![Maven Package upon a push](https://github.com/mosip/keymanager/actions/workflows/push_trigger.yml/badge.svg?branch=develop)](https://github.com/mosip/keymanager/actions/workflows/push_trigger.yml)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?branch=master&project=mosip_keymanager&metric=alert_status)](https://sonarcloud.io/dashboard?branch=develop&id=mosip_keymanager)
+[![Maven Package upon a push](https://github.com/mosip/keymanager/actions/workflows/push_trigger.yml/badge.svg?branch=release-1.2.0.1)](https://github.com/mosip/keymanager/actions/workflows/push_trigger.yml)
```
Comment (ckm007) [inline README.md:1]: branch name here will be develop.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?branch=release-1.2.0.1&project=mosip_keymanager&metric=alert_status)](https://sonarcloud.io/dashboard?branch=release-1.2.0.1&id=mosip_keymanager)
```
Comment (ckm007) [inline README.md:2]: branch name here will be develop.

File: README.md (hunk 1 above)
Comment (mahammedtaheer) [reply README.md]: changed the branch name to develop

File: README.md (hunk 2 above)
Comment (mahammedtaheer) [reply README.md]: changed the branch name to develop

**Review by ckm007 — APPROVED** [review]

**Pull request merged**